ctapipe/io/dl1eventsource.py

```diff
@@ -0,0 +1,389 @@
+from ctapipe.io.datalevels import DataLevel
+from ctapipe.instrument import (
+    TelescopeDescription,
+    SubarrayDescription,
+    CameraGeometry,
+    OpticsDescription,
+)
+from ctapipe.containers import EventAndMonDataContainer, EventType
+from ctapipe.containers import MCHeaderContainer, MCEventContainer
+from ctapipe.io.eventsource import EventSource
+from astropy.coordinates import Angle
+import astropy.units as u
+import tables
+import warnings
+
+
+class DL1EventSource(EventSource):
+    def __init__(
+        self,
+        input_url,
+        config=None,
+        parent=None,
+        **kwargs
+    ):
+        """
+        EventSource for dl1 files in the standard DL1 data format
+
+        Parameters:
+        -----------
+        input_url : str
+            Path of the file to load
+        config: ??
+        parent: ??
+        kwargs
+        """
+        super().__init__(
+            input_url=input_url,
+            config=config,
+            parent=parent,
+            **kwargs
+        )
+
+        self.file_ = tables.open_file(input_url)
+        self._subarray_info = self._prepare_subarray_info(
+            self.file_.root.configuration.instrument
+        )
+        self._mc_header = self._parse_mc_header(
+            self.file_.root.configuration.simulation.run
+        )
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.close()
+
+    def close(self):
+        self.file_.close()
+
+    @staticmethod
+    def is_compatible(file_path):
+        """
+        Implementation needed!
+        """
+        return False
```

Comment:
The first 8 bytes of all HDF5 files are `b'\x89HDF\r\n\x1a\n'`, which is an easy way of make sure it's hdf5.
Then you could check for the metadata we write in stage1.

Reply:
https://www.loc.gov/preservation/digital/formats/fdd/fdd000229.shtml

Reply:
So the first step would be:

```
with open(file_path, 'rb') as f:
     magic_number = f.read(8)
if magic_number != b'\x89HDF\r\n\x1a\n':
    return False
```